src/main/java/mate/academy/bookshop/dto/shoppingcart/UpdateCartItemRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookshop.dto.shoppingcart;
+
+import jakarta.validation.constraints.Min;
+
+public record UpdateCartItemRequestDto(
+        @Min(0)
```

Comment:
bacause if we want to remove that item, we'd use another endpoint to remove the whole item entity from the list of the cart, and not set 0 for it
```suggestion
        @Min(1)
```